build-release.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+import shutil
+import subprocess
+import argparse
+import subprocess
+from pathlib import Path
+
+FRONTEND_IMAGE_NAME = "planttracker-frontend"
+FRONTEND_PATH_INSIDE_CONTAINER = "/app/dist"
+
+def execute_command(command: list[str]) -> None:
+    """Execute given command in terminal"""
+    subprocess.run(command, check=True)
+
+def build_frontend(output_path: Path) -> None:
+    """Build the frontend using the Docker image, creating temporary container and extracting final image into provided path."""
+    execute_command(["docker", "build", "-t", FRONTEND_IMAGE_NAME, "frontend"])
+    # Get the container ID of the temporary container
+    result = subprocess.run(["docker", "create", FRONTEND_IMAGE_NAME], check=True, text=True, capture_output=True)
+    container_id = result.stdout.strip()
+    execute_command(["docker", "cp", f'{container_id}:{FRONTEND_PATH_INSIDE_CONTAINER}', str(output_path)])
+    execute_command(["docker", "rm", "-v", container_id])
+
+def build_backend(output_path: Path) -> None:
+    """Build the backend using Cargo"""
+    execute_command(["cargo", "build", "--release"])
+    execute_command(["cp", "target/release/planttracker-backend", str(output_path)])
+
+def main() -> None:
+    """Entry point of the script."""
+    # Parse command line arguments
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--output-path", help="Destination path to copy the app", type=Path)
```

Comment:
I think it could be a good idea to put here some default path, which will correspond to the path within .env.dev. To make the installation process much straight forward - for the beginning user will not need to modify any parameters.